SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The Drand team actively supports the most recent minor version of drand.
+Upcoming major or minor releases are announced with a prerelease at least one week in advance, and are communicated through the leagueofentropy@googlegroups.com email list.
```

Comment:
Do we really have to tie our hands by saying this publicly ? If there's an urgent patch, I wouldn't want to wait one week. Usually, ppl release new versions and then tell the world and there can be a migration plan etc.
Im not sure I see the necessity of a warning _in advance_

Reply:
once we have clients using the client library, we need to let them know in advance if a release fixes a security issue, especially if protocol breaking. I think that's the main point here.

Reply:
@willscott is right here. Doing proper release management is part of making a software library production ready.

See the level of detail that go-ipfs goes into about their release process at -- https://github.com/ipfs/go-ipfs/blob/master/docs/releases.md --.

We should not tie the release process into a specific mailing list of adopters, but craft a release process that is public (e.g. Github issue) and then treat the LoE as one of the adopters (equivalent to stage 3 from https://github.com/ipfs/go-ipfs/blob/master/docs/releases.md#stage-3---community-prod-testing).

Reply:
Ok sounds good - and yes for LoE -we should post it on drand.love or stg

Reply:
where are we going to communicate this? if loe email list pointed here isn't publicly joinable, it doesn't seem like a great announcement channel.